Stories from Epics in Studio Pro (#5105)
mendix/docs: content/en/docs/refguide/modeling/menus/view-menu/stories-pane.md

## 1 Introduction 

For version control enabled apps (apps with Team Server or other SVN servers), the **Stories** pane displays the stories and tasks of the current Sprint. Stories, tasks, and Sprints are created and managed via [Stories](/developerportal/collaborate/stories/) or [Epics](/developerportal/collaborate/epics/) in the Developer Portal.

In the **Stories** pane, you can view the stories and tasks of the current Sprint, show or hide done stories, open their details, change the status of the stories and tasks, and open documents connected to stories. 

> {{% alert color="info" %}}If you use Stories, both stories and tasks of the current Sprint will be shown in the **Stories** pane. For more information on tasks, see the [Adding a Task to a Story](/developerportal/collaborate/stories/#adding-task) section in *Stories*.

Reviewer: ```suggestion
{{% alert color="info" %}}If you use Stories, both stories and tasks of the current Sprint are shown in the **Stories** pane. For more information on tasks, see the [Adding a Task to a Story](/developerportal/collaborate/stories/#adding-task) section in *Stories*.
```

> If you use Epics, only the stories of the current Sprint will be shown in the **Stories** pane; the tasks of each story will not be shown. You also cannot change the status of the stories in the **Stories** pane.{{% /alert %}}

Reviewer: ```suggestion
If you use Epics, only the stories of the current Sprint are shown in the **Stories** pane; the tasks of each story are not displayed. You also cannot change the status of the stories in the **Stories** pane.{{% /alert %}}
```

### 2.5 Changing the Status of a Story or a Task {#changing-status}

> If you use Stories in the Developer Portal for this app, you can change the status of a story or a task on the **Stories** pane.

Reviewer: ```suggestion
If you use Stories in the Developer Portal for this app, you can change the status of a story or a task on the **Stories** pane. If you use Epics, you cannot change the status of the stories in the pane. 
```
